docs/2-the-zarf-cli/2-declarative-cli-configuration.md

```diff
@@ -0,0 +1,48 @@
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import ExampleYAML from '@site/src/components/ExampleYAML';
+import FetchFileCodeBlock from '@site/src/components/FetchFileCodeBlock';
+
+# Declarative CLI Configuration
```

Comment:
```suggestion
# Zarf Config Files
```
Not sure if this is the best wording (may be good to get more feedback from others) but I personally find declarative to be weird to use in a lot of places - config files do help with declaring configuration but they aren't the only way to do so - also most of our left sidebar titles are more concrete things (package components, zarf package schema) rather than philosophical concepts.